.github/workflows/automated-tests.yml

```diff
@@ -0,0 +1,28 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Run tests
+
+# Controls when the action will run. 
+on: push
+
+jobs:
+  tests:
+    name: Automated tests
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Macos test seem to stuck for some reason.. Make sense to try after changes in https://github.com/browser-automation/cba-components/pull/55 gets merged.